runtime/v2/bundle.go

```diff
@@ -115,16 +116,24 @@ type Bundle struct {
 }
 
 // Delete a bundle atomically
-func (b *Bundle) Delete() error {
+func (b *Bundle) Delete(ctx context.Context) error {
 	work, werr := os.Readlink(filepath.Join(b.Path, "work"))
 	rootfs := filepath.Join(b.Path, "rootfs")
+
+	// on windows hcsshim writes panic logs in the bundle directory in a file named
+	// "panic.log" log those messages (if any).
+	logs, err := ioutil.ReadFile(filepath.Join(b.Path, "panic.log"))
```

Comment:
Should we be defensive about how much data we'll read from this file?

Reply:
That's a good point. Maybe we can read only first few KBs of that file and log that? That should cover all of the actual panic logs.

Reply:
panic.log just contains the stack of where the panic originated from right?

Reply:
I remember Kevin had looked into a semi-related issue where when we were dumping all of the running goroutine stacks and trying to log this, it hit etws maximum size limit. He [made a change](https://github.com/containerd/containerd/pull/4787) to split them on new lines to avoid this. If panic.log just contains a single goroutines stack from where the panic originated from I don't think it's likely we'd hit this though.

Reply:
Doesn't anything written to stderr in the shim process end up in panic.log?

Reply:
No you're right Scott. I don't think we actually log anything to the std handles though, the only time would be for a panic unless I'm way off. https://github.com/microsoft/hcsshim/blob/2d5a2c3c03768113f35f75172d0db295f72fd31d/cmd/containerd-shim-runhcs-v1/serve.go#L64